### DISCUSSION TITLE
Getting Error affects Humans output

### DISCUSSION THREAD

**Discussion by YaraAmin** [vladmandic/human#192] [Q&A] [answered]
Hi
I'm getting this error sometimes: `Uncaught (in promise) DOMException: Failed to read the 'cssRules' property from 'CSSStyleSheet': Cannot access rules` and then the output of human has a lot of nulls. this happened when I used the video stream and when I used a screenshot of the stream it only showed when I'm disposing the frame after the detection.

**Comment by vladmandic** [comment, marked as answer]
As a security measure, Chrome enforces **CORS** (cross-origin resource sharing) rules so  
nobody can access DOM element properties directly  

You need to set appropriate HTTP header on your web server if you want to access property directly  

If you're using `human` built-in dev server, it sets it for you  
If you're using your own (or using `NextJS` which does it for you), you have to configure CORS rules  
See <https://enable-cors.org/server.html> for examples

**Comment by YaraAmin** [comment]
Thanks, I believe the issue is in disposing the objects (frames, tensor, output) I tried it using human.tf.dispose and using tf.dispose and still getting the same error

**Reply by vladmandic** [reply]
Why would you think that? Can you explain in a bit more details what's going on?
